### PR TITLE
Feature: Expand space in the title bars to account for longer names

### LINKED ISSUE #1082: Expand space available for Flow name
**Why**
User from community reported frustration on the Flow > Flow Run page specifically. When the name of the flow is too long, it correctly truncates, however it appears that there is a lot of wasted space that we are not utilizing. 

image

**What**

- [ ] Expand available space for flow name 
- [ ] Validate that other "title bars" also utilize their available space appropriately for smaller screens

**Notes**
https://prefect-community.slack.com/archives/C0192RWGJQH/p1634005499335900

### PR DESCRIPTION
**Pull request by ThatGalNatalie**
PR Checklist:

- [x] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
Should resolve #1082

### CODE REVIEW

File: src/pages/Flow/Flow.vue [hunk 1]
```diff
@@ -261,7 +261,12 @@ export default {
             : $vuetify.breakpoint.smAndDown && { display: 'inline' }
         "
       >
-        <div v-if="flowGroup">
+        <div
+          v-if="flowGroup"
+          class="text-truncate"
+          style="max-width: 65vw"
+          :style="{ width: selectedFlow.name.length + 'ch' }"
```
[the remark this answers is not in the document]
Comment (ThatGalNatalie) [reply src/pages/Flow/Flow.vue:268]: thanks!

**Review by zhen0 — DISMISSED** [review]
This looks good on larger screens but as discussed, I think you'll need to consider using breakpoints here.  The name is interfering with action buttons on smaller screens.  Happy to pair with you on this one if needed. 

Larger screen:

Screen Shot 2021-11-01 at 2 35 41 PM

Smaller:
Screen Shot 2021-11-01 at 2 33 32 PM

Mobile:
Screen Shot 2021-11-01 at 2 33 08 PM

Current Staging - larger screen:



Screen Shot 2021-11-01 at 2 40 26 PM

**Comment by zhen0** [comment]
Screen Shot 2021-11-05 at 4 35 11 PM
Screen Shot 2021-11-05 at 4 35 20 PM

**Review by zhen0 — DISMISSED** [review]
Thanks @ThatGalNatalie - those updated views look good!

**Comment by zhen0** [comment]
I'm getting issues from the changelog @ThatGalNatalie.   Can you remove the changelog entry and update the PR title?

**Pull request merged**